packages/stable/src/api/3d/assetMappings3DApi.ts

```diff
@@ -30,6 +38,30 @@ export class AssetMappings3DAPI extends BaseResourceAPI<AssetMapping3D> {
     );
   };
 
+  /**
+   * [Filter 3D asset mappings](https://docs.cognite.com/api/v1/#operation/filter3DAssetMappings)
+   *
+   * ```js
+   * const mappings3D = await client.assetMappings3D.filter(3244265346345, 32423454353545, {
+   *   filter: {
+   *     treeIndexes: [1000, 1001, 1002]
+   *   }
+   * });
+   * ```
+   */
+  public filter = (
+    modelId: CogniteInternalId,
+    revisionId: CogniteInternalId,
+    query: Filter3DAssetMappingsQuery
+  ): CursorAndAsyncIterator<AssetMapping3D> => {
+    const path = `${this.encodeUrl(modelId, revisionId)}/list`;
+    return super.listEndpoint(
+      params =>
+        this.post<CursorResponse<AssetMapping3D[]>>(path, { data: params }),
+      query
+    );
```

Comment:
lines 58-62 above should be possible to replace with
```
super.listEndpoint(this.callListEndpointWithPost, query);
```

Reply:
Would that call use the right URL? The URL must contain `modelId` and `revisionId`, which must be communicated somehow.

Reply:
yes, you're right. then disregard my previous comment